### PR TITLE
Add customizable ok and cancel texts for localization support.

### LINKED ISSUE #60: Add customizable ok and cancel texts
These texts are currently hardcoded. They should be customizable to allow localization of the component.

### PR DESCRIPTION
**Pull request by panuhorsmalahti**
Fixes #60

### CODE REVIEW

File: app-datepicker-dialog.html [hunk 1]
```diff
@@ -77,6 +77,14 @@
         pageEntryAnimation: String,
         pageExitAnimation: String,
         locale: String,
+        ok: {
```
Comment (motss) [inline app-datepicker-dialog.html]: @panuhorsmalahti Maybe a better naming? How about `confirmLabel`? WDYT?

Comment (panuhorsmalahti) [inline app-datepicker-dialog.html]: Sure, sounds good, I'll change it.

File: app-datepicker-dialog.html [hunk 2]
```diff
@@ -77,6 +77,14 @@
...
+          type: String,
+          value: "ok"
+        },
+        cancel: {
```
Comment (panuhorsmalahti) [inline app-datepicker-dialog.html]: Sure, sounds good, I'll change it.

**Comment by panuhorsmalahti** [comment]
Fixed the attribute names.

**Comment by motss** [comment]
@panuhorsmalahti Cool. LGTM. Thank you for the contribution.

**Pull request merged**